contracts/lbp/LBPManager.sol

```diff
@@ -0,0 +1,290 @@
+/*
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+*/
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
This PR states that it closes #132 . Can you pls help me here, where can i find states described in #132 ?

Reply:
This ticket should have been closed earlier. I took what was decided in the workshops by Doug, that we need states, and transferred it to an issue. In the comments of the issue, Mihir commented that these states are needed for the frontend but should not reflect contract states. After that, this issue could have been closed

Reply:
Hmm, ok, it is still kinda closed in the description of this PR, can you pls remove it then and close it manually?

Reply:
Done